src/tests/ftest/util/telemetry_utils.py

```diff
@@ -511,6 +511,12 @@ class TelemetryUtils():
         ENGINE_NVME_RELIABILITY_METRICS +\
         ENGINE_NVME_CRIT_WARN_METRICS +\
         ENGINE_NVME_INTEL_VENDOR_METRICS
+    ENGINE_MEM_USAGE_METRICS = [
+        "engine_mem_vos_dtx_cmt_ent_48",
+        "engine_mem_vos_vos_obj_360",
+        "engine_mem_vos_vos_lru_size",
+        "engine_mem_dtx_dtx_leader_handle_336",
+        "engine_mem_dtx_dtx_entry_40"]
```

Comment:
 What are the postfix digits used for?

Reply:
If we are going to use the size like this, it may be better to have a macro

#define metric_from_struct(name) "engine_mem_" #name "_" stringify(sizeof(struct name))

Or some such so it's always right.

My preference would be to leave the size out and add _cnt or something at end.